web/templates/settings/notification_settings.hbs

```diff
@@ -70,7 +70,7 @@
               prefix=../prefix}}
         {{/each}}
 
-        <label for="notification_sound">
+        <label for="{{prefix}}notification_sound">
```

Comment:
Ahh, yeah, this likely fixes a bug with the "default notification settings" panel.